packages/strapi-helper-plugin/lib/src/components/ErrorFallback/index.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+// https://github.com/bvaughn/react-error-boundary#usage
+
+function ErrorFallback({ error /* resetErrorBoundary */ }) {
+  return (
+    <div style={{ background: '#ffff' }}>
+      <h2>Something went wrong.</h2>
+      <details style={{ whiteSpace: 'pre-wrap' }}>
+        {error.message}
+        <br />
+        {error.stack}
+      </details>
+    </div>
+  );
+}
+
+ErrorFallback.defaultProps = {
+  error: { message: null },
+  // resetErrorBoundary: PropTypes.func,
```

Comment:
you can remove this one if not necessary

Reply:
The library I have added supports such propTypes I felt it was good to comment it so when we start improving the error management we know such option is available.

WDYT of keeping the comment?

Reply:
Not a huge fan of predicting the future (personal opinion). feel free to keep it if you think it's useful 😊

Reply:
Let's remove it, I have added the link to the library's documentation in the component's file anyway.